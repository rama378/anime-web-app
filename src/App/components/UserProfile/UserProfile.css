.UserProfile {}

.UserProfile__user {
  display: inline-block;
  margin-right: 16px;
  font-weight: bold;
}
