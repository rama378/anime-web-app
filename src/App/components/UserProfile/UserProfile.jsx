import './UserProfile.css';
import {ReactComponent as UserIcon} from "../../common/assets/images/icon/user.svg";
import Button from '../../common/components/Button/Button';

const UserProfile = () => {
	return (
		<div>
			<div className='UserProfile__user'>Name</div>
			<Button>
				<UserIcon />
			</Button>
		</div>
	);
};

export default UserProfile;
