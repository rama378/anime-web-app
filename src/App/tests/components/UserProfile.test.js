import React from 'react';
import ShallowRenderer from 'react-test-renderer/shallow';

import UserProfile from '../../components/UserProfile/UserProfile';

const renderer = new ShallowRenderer();

describe('<UserProfile />', () => {
  it('should render and match the snapshot', () => {
    renderer.render(<UserProfile />);
    const renderedOutput = renderer.getRenderOutput();
    expect(renderedOutput).toMatchSnapshot();
  });
});
