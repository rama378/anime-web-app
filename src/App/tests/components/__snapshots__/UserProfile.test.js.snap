// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<UserProfile /> should render and match the snapshot 1`] = `
<div>
  <div
    className="UserProfile__user"
  >
    Name
  </div>
  <Button
    style="default"
  >
    <ForwardRef(SvgUser) />
  </Button>
</div>
`;
